src/components/header/Header.component.tsx

```diff
@@ -32,68 +39,78 @@ const Header: React.FC<HeaderProps> = ({ onShowAbout, onShowProducts }) => (
                                         strokeLinejoin="round"
                                     />
                                 </svg>
-                            </svg>
-                        </button>
-                        <img className={styles.header__theme_icon} src={headerThemeElement} alt="Element theme" />
-                        <button className={styles.header__theme_button_dark}>
-                            <svg width="30" height="34" viewBox="0 0 30 34" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            </button>
+                            <img className={styles.header__theme_icon} src={headerThemeElement} alt="Element theme" />
+                            <button
+                                className={styles.header__theme_button_dark}
+                                onClick={showTheme}
+                                style={{ color: theme === 'dark' ? '#FFF' : '#656565' }}
+                            >
+                                <svg width="30" height="34" viewBox="0 0 30 34" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path
+                                        d="M12.5 12.0001C12.5 16.1422 15.8579 19.5001 20 19.5001C20.7577 19.5001 21.4892 19.388 22.1787 19.179C21.2453 22.2586 18.3844 24.5 15 24.5C10.8579 24.5 7.5 21.1423 7.5 17.0002C7.5 13.6158 9.74173 10.7549 12.8213 9.82153C12.6124 10.511 12.5 11.2424 12.5 12.0001Z"
+                                        stroke="currentColor"
+                                        strokeWidth="1.5"
+                                        strokeLinecap="round"
+                                        strokeLinejoin="round"
+                                    />
+                                </svg>
+                            </button>
+                        </div>
+                    </div>
+
+                    <nav className={styles.header__navigation}>
+                        <ul className={styles.navigation__list}>
+                            <li className={styles.navigation__item}>
+                                <button
+                                    onClick={onShowAbout}
+                                    className={styles.navigation__link}
+                                    style={{ fontWeight: isShowPage === 'About' ? 'bold' : 'normal' }}
+                                >
+                                    About
+                                </button>
+                            </li>
+                            <li className={styles.navigation__item}>
+                                <button
+                                    onClick={onShowProducts}
+                                    className={styles.navigation__link}
+                                    style={{ fontWeight: isShowPage === 'ProductsList' ? 'bold' : 'normal' }}
```

Comment:
You should use classes instead of "style line"
style={{ fontWeight: isShowPage === 'ProductsList' ? 'bold' : 'normal' }}